Process "/home/david/Projects/rst2pdf_py2/bin/rst2pdf" started on Fri Jun  5 12:30:36 2015

        /home/david/Projects/rst2pdf_py2/bin/rst2pdf --date-invariant
        -v test_png_dpi.txt -o
        /home/david/Projects/rst2pdf_py2/rst2pdf/tests/output/test_png_dpi.pdf


* [INFO] createpdf.py:1572 Importing extension module 'vectorpdf'
* [INFO] languages.py:39 Language 'en_US' not supported by Docutils, using 'en' instead.
* [INFO] styles.py:82 Using stylesheets: 
* [INFO] styles.py:110 FontPath:['/home/david/Projects/rst2pdf_py2/rst2pdf/fonts', '.']
* [INFO] styles.py:111 StylePath:['/home/david/Projects/rst2pdf_py2/rst2pdf/styles', '/home/david/.rst2pdf/styles', '.']
* [INFO] languages.py:39 Language 'en_US' not supported by Docutils, using 'en' instead.
* [ERROR] image.py:357 The image (/home/david/Projects/rst2pdf_py2/rst2pdf/tests/input/test_png_dpi_1.png, near line 2 in file test_png_dpi.txt) is broken or in an unknown format
* [ERROR] image.py:357 The image (/home/david/Projects/rst2pdf_py2/rst2pdf/tests/input/test_png_dpi_2.png, near line 3 in file test_png_dpi.txt) is broken or in an unknown format
* [INFO] createpdf.py:651 Starting build
* [INFO] createpdf.py:1112 Page 1 [1]

Program /home/david/Projects/rst2pdf_py2/bin/rst2pdf exit code: PASS (0)   elapsed time: 00.2 seconds

Validity of file test_png_dpi.pdf checksum '98c65ea48a7f48b131a24e133961c061' is unknown.
